lib/apis/LWA.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const ApiBase = require('./ApiBase');
+
+class LWA extends ApiBase {
+
+  // eslint-disable-next-line class-methods-use-this
+  getToken() {
+    return undefined;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  getEndpoint() {
+    return 'https://api.amazon.com';
+  }
+
+  /*
+   * Gets the user's profile information.
+   * https://developer.amazon.com/docs/login-with-amazon/customer-profile.html
+   */
+  getUserInformation() {
+    const accessToken = this.alexaEvent.user.accessToken;
```

Comment:
shouldn't this come from `getToken`?

Reply:
`getToken` provides a token for the Authorization headers. LWA adds it as a GET parameter :/